servicex_app/servicex/resources/internal/add_file_to_dataset.py

```diff
@@ -61,8 +61,11 @@ def put(self, request_id):
                 self.lookup_result_processor.add_file_to_dataset(submitted_request, db_record)
 
             db.session.commit()
-            current_app.logger.info(f"Got {len(add_file_request)} files.",
-                                    extra={'requestId': request_id})
+            current_app.logger.info("Adding files.",
+                                    extra={
+                                        'requestId': request_id,
+                                        'nfiles': len(add_file_request)
```

Comment:
I don't think the nfiles will get included in the log output

Reply:
it won't. But I don't want any of the extra things included. It does get sent to logstash where it matters.

Reply:
Okay, but what about admins not using logstash?  We need to also consider what admins that are only going to get the console output need in order to debug problems.  I'd rather not have to start spinning up a special version of the container with logging changes in order to debug an issue at BNL or UNL or with Burt's instance.

Reply:
I have added extra fields in output. 
but I would really prefer if people would just leave the default settings and use ES to debug things. It is much easier when you can for example select a requestID or source file and get only these lines.